### PR TITLE
images/k8s-infra: Add rclone

### PR DESCRIPTION
**Pull request by ameukam**
- Install [rclone](https://rclone.org/)
- remove usage of apt-key for package installation

Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

**Comment by ameukam** [comment]
cc @BenTheElder

File: images/k8s-infra/cloudbuild.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ steps:
     - TAG=$_TAG
     - TAG_DATE_SHA=$_GIT_TAG
     - _=$_PULL_BASE_REF # unused
-serviceAccount: 'projects/$PROJECT_ID/serviceAccounts/gcb-image-builder@$PROJECT_ID.iam.gserviceaccount.com'
```
Comment (BenTheElder) [inline images/k8s-infra/cloudbuild.yaml:14]: ? unrelated ?

Comment (ameukam) [reply images/k8s-infra/cloudbuild.yaml:14]: Added in https://github.com/kubernetes/k8s.io/pull/4028 to test GCB with dedicated service accounts. Not impactful for the PR and the jobs using this image.
